src/instructlab/server.py

```diff
@@ -151,6 +153,17 @@ def ensure_server(
         return (server_process, temp_api_base, queue)
 
 
+def is_valid_model_path(logger, model_path):
+    model = Path(model_path)
+    if not model.is_file():
+        logger.error(f"Model file does not exist: '{model_path}'")
+        return False
+    if model.suffix != ".gguf":
```

Comment:
We have other potential acceptable model types. Safetensors are usually a directory. I think gguf and safe tensors are the main types we deal with now. I think this extra check isn't necessary - the user provided a path, the path could be to a non-gguf model, here we just indicate the path the user put in config.yaml exists or not. I would leave logic about whether or not that path, if it exists, points to a legit model file up to the model serving backend in use (here it's llama.cpp.) Does that make sense?